packages/dicom/typescript/index.html

```diff
@@ -77,7 +76,7 @@
       </section>
     </sp-theme>
 
-    <script type="module" src="./dist/itk-dicom.js"></script>
-    <script type="module" src="./dist/demo/app.js"></script>
+    <script type="module" src="./dist/bundles/itk-dicom.js"></script>
+    <script type="module" src="./test/browser/app.ts"></script>
```

Comment:
We are changing `dist/demo/app.js` to `test/browser/app.ts` but this is a demo and not an automated browser test. Is this really correct?

Reply:
It is both a demo and the basis for tests with Cypress.